src/fragments/lib/graphqlapi/js/authz.mdx

```diff
@@ -40,27 +65,82 @@ You can implement your own custom API authorization logic using an AWS Lambda fu
 If you are using a Lambda function as an authorization mode with your AppSync API, you will need to pass an authentication token with each API request and will need to manage token refresh in your application.
 
 The following example assumes `AWS_LAMBDA` is configured as the default authentication type for your API:
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+// ...
```

Comment:
I see a few of these ellipses "headers" were added in the latest rev as if to acknowledge "all the normal imports and configs happen here". But, I'm not sure I see the need. The rest of our docs seem content to assume imports and context are implied in each example. Is there a reason we're breaking with that pattern in this PR?

Reply:
Chatted about this offline. Resolving